### PR TITLE
Show Common-backing language and prevent it from double-counting and displaying separately

### LINKED ISSUE #13675: Languages: knowing both Common and the actual language count as two languages to the character's max
Assume the Common language is set to Taldane for argument's sake

If a character picks Common and Taldane, it counts as if it knew two languages.
While not technically a bug, it would be nice if it didn't count so.

I'd also like to take the opportunity to suggest that:

- The Common language's name gets appended to Common, similar to how it is displayed on the language selector.
- The actual language gets hidden from the list if Common is known to avoid confusion when counting the number of known languages.

### PR DESCRIPTION
**Pull request by DocSchlock**
Closes #13675

GMs can still alter the common-linked language on PC sheets in case it becomes needed.

### CODE REVIEW

**Comment by stwlam** [comment]
I'm not sure that this is the most intuitive solution, and the actual issue can be addressed just by not counting both Common:tm: and the language that equates to Common:tm: when tallying number of languages known.

**Comment by DocSchlock** [comment]
@stwlam 

Changed it to show the backing language as a tooltip instead and reworked the logic to not count the Common-backing language against the total language slots in the sheet.

File: src/module/actor/character/sheet.ts [hunk 1]
```diff
@@ -315,25 +315,48 @@ class CharacterSheetPF2e<TActor extends CharacterPF2e> extends CreatureSheetPF2e
...
+        const commonLanguage = game.settings.get("pf2e", "homebrew.languageRarities").commonLanguage;
+
         sheetData.languages = ((): LanguageSheetData[] => {
             const languagesBuild = actor.system.build.languages;
             const sourceLanguages = actor._source.system.details.languages.value.filter(
                 (l) => l in CONFIG.PF2E.languages,
             );
-            const isOverMax = languagesBuild.value > languagesBuild.max;
+
+            const deductBackingLanguage = commonLanguage
```
Comment (stwlam) [inline src/module/actor/character/sheet.ts:327]: This is (or should be) guaranteed to be present

Comment (CarlosFdez) [reply src/module/actor/character/sheet.ts]: Even if its not, the following logic will still work out fine, the actor wouldn't have a common language of null.

Comment (DocSchlock) [reply src/module/actor/character/sheet.ts]: Bad variable name there. I refactored it - it's supposed to be the backing language.

File: src/module/actor/character/sheet.ts [hunk 2]
```diff
@@ -315,23 +315,50 @@ class CharacterSheetPF2e<TActor extends CharacterPF2e> extends CreatureSheetPF2e
...
         sheetData.languages = ((): LanguageSheetData[] => {
             const languagesBuild = actor.system.build.languages;
             const sourceLanguages = actor._source.system.details.languages.value.filter(
                 (l) => l in CONFIG.PF2E.languages,
             );
-            const isOverMax = languagesBuild.value > languagesBuild.max;
+
+            // find if the backing language is present in character's languages
+            // this means characters explictly selected it
+            const deductBackingLanguage = commonBackingLanguage
```
Comment (stwlam) [inline src/module/actor/character/sheet.ts:329]: Same

File: src/module/actor/character/sheet.ts [hunk 3]
```diff
@@ -315,23 +315,50 @@ class CharacterSheetPF2e<TActor extends CharacterPF2e> extends CreatureSheetPF2e
 
         sheetData.abpEnabled = AutomaticBonusProgression.isEnabled(actor);
 
+        // get the backing language for common
+        const commonBackingLanguage = game.settings.get("pf2e", "homebrew.languageRarities").commonLanguage;
```
Comment (stwlam) [inline src/module/actor/character/sheet.ts:319]: Retrieve from cache

File: src/module/actor/character/sheet.ts (hunk 2 above)
Comment (DocSchlock) [reply src/module/actor/character/sheet.ts]: Cleaned up

File: src/module/actor/character/sheet.ts (hunk 3 above)
Comment (DocSchlock) [reply src/module/actor/character/sheet.ts]: Added to cache and called from there.

**Comment by stwlam** [comment]
Organizing this a little differently so that actor data isn't being calculated in a sheet class

**Pull request closed without merging**